pkgs/by-name/di/dircolors-solarized/package.nix

```
@@ -0,0 +1,34 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+  unstableGitUpdater,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dircolors-solarized";
+  version = "52bfa164e4388ee232f6a9235f62e8482e1f1767";
+
+  src = fetchFromGitHub {
+    owner = "seebi";
+    repo = pname;
+    rev = version;
```

(you'll need to setup `finalAttrs`)
```suggestion
    rev = finalAttrs.version;
```